RELEASES.md

```diff
@@ -0,0 +1,248 @@
+
+__If you are reporting a security issue, please follow the responsible
+disclosure guidelines and reach out discretely__. Remember that backported PRs
+must follow the versioning guidelines from this document.
+
+Any release that is "active" can accept backports. Opening a backport PR is
+fairly straightforward. The steps differ depending on whether you are pulling
+a fix from master or need to draft a new commit specific to a particular
+branch.
+
+To cherry pick a straightforward commit from master, simply use the cherry pick
+process:
+
+1. Pick the branch to which you want backported, usually in the format
+   `release/<minor>.<major>`. The following will create a branch you can
+   use to open a PR:
+
+	```console
+	$ git checkout -b my-backport-branch release/<major>.<minor>.
+	```
+
+2. Find the commit you want backported.
+3. Apply it to the release branch:
+
+	```console
+	$ git cherry-pick -xsS <commit>
+	```
+4. Push the branch and open up a PR against the _release branch_:
+
+	```
+	$ git push -u stevvooe my-backport-branch
+	```
+
+   Make sure to replace `stevvooe` with whatever fork you are using to open
+   the PR. When you open the PR, make sure to switch `master` with whatever
+   release branch you are targeting with the fix.
+
+If there is no existing fix in master, you should first fix the bug in master,
+or ask us a maintainer or contributor to do it via an issue. Once that PR is
+completed, open a PR using the process above.
+
+Only when the bug is not seen in master and must be made for the specific
+release branch should you open a PR with new code.
+
+## Public API Stability
+
+The following table provides an overview the components covered by containerd
+versions:
+
+
+| Component  | Status   | Stablized Version | Links         |
+|------------|----------|-------------------|---------------|
+| GRPC API   | Alpha    | 1.0               | [api/](api) |
+| Go API     | Unstable | 1.1 tentative     | [godoc](https://godoc.org/github.com/containerd/containerd) |
+| `ctr` tool | Unstable | Out of scope      | -             |
+
+From the version stated in the above table, that component must adhere to the
+stability constraints expected in release versions.
+
+Unless explicitly stated here, components that are called out as unstable or
+not covered may change in a future minor version. Breaking changes to
+"unstable" components will be avoided in patch versions.
+
+### GRPC API
+
+The primary product of containerd is the GRPC API. As of the 1.0.0 release, the
+GRPC API will not have any backwards incompatible changes without a _major_
+version jump.
+
+To ensure compatibility, we have collected the entire GRPC API symbol set into
+a single file. At each minor release of containerd, we will move the current
+`next.pb.txt` file to a file named for the minor version, such as `1.0.pb.txt`,
+enumerating the support services and messages. See [api/](api) for details.
+
+Note that new services may be added in _minor_ releases.
```

Comment:
What else can be added in a _minor_ release?  Would a _minor_ release also allow the addition of service calls, parameters, and return elements?